test/e2e/config/default.json

```diff
@@ -19,6 +19,31 @@
 	"reportWarningsToSlack": false,
 	"closeBrowserOnComplete": true,
 	"highlightElements": false,
+	"artifacts": {
+		"screenshot": "screenshots",
+		"video": "screenshots/videos",
+		"log": "."
+	},
+	"displaySize": "desktop",
+	"displayResolutions": {
+		"mobile": {
+			"width": 400,
+			"height": 1000
+		},
+		"tablet": {
+			"width": 1024,
+			"height": 1000
+		},
+		"desktop": {
+			"width": 1440,
+			"height": 1000
+		},
+		"laptop": {
+			"width": 1440,
+			"height": 790
+		}
+	},
+	"locale": "en",
```

Comment:
Having those values here is much easier to follow, thanks! 👍

Reply:
You didn't explicitly mention it here, but I think it's a fair assumption that terminology change (viewport, etc) should also apply here as well?